### ISSUE TITLE
overrides-dsl repo link broken

### ISSUE THREAD

**Issue by langston-barrett** [GaloisInc/ambient-verifier#1]
This link in the README appears broken: https://gitlab-ext.galois.com/ambient/overrides-dsl

**Comment by RyanGlScott** [comment]
Good catch. That portion of `ambient-verifier` isn't usable at the moment, so we should just remove it for the time being.
